keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -383,6 +469,8 @@ Describe the metrics themselves and the reasons why they weren't added (e.g., co
 implementation difficulties, etc.).
 -->
 
+Yes. It would be useful if we could see more details related to scheduler's decisions in metrics.
```

Comment:
Link to the issue.

Maybe be more specific about decisions. In this case, we would like to know which Filters had impact on the scheduling of the pod.